src/data/faq_de.json

```
@@ -569,6 +569,41 @@
                     }
                 ]
             },
+            {
+                "title": "Zertifikatprüfung bei Buchungen",
+                "id": "val_service",
+                "accordion": [
+                    {
+                        "title": "Warum erhalte ich die Meldung, dass kein geeignetes Zertifikat vorhanden ist?",
+                        "anchor": "val_service_no_valid_dcc",
+                        "active": true,
+                        "textblock": [
+                            "Die App überprüft, ob Ihre Zertifikate den Anforderungen des Buchungssystems entsprechen. Dabei werden verschiedene Parameter geprüft. Zunächst muss der Typ Ihres Zertifikats den Anforderungen des Buchungssystems entsprechen. Informieren Sie sich hierzu über die akzeptierten Zertifikate bei Ihrem Ticketanbieter, fügen Sie die entsprechenden Zertifikate in der App hinzu, sofern noch nicht geschehen und stellen Sie sicher, dass Ihre Zertifikate gültig sind.",
+                            "<b>Was sollten Sie beachten?</b>",
+                            "Sie sollten unbedingt darauf achten, dass der Name, der im Zertifikat, das sie zur Überprüfung benutzen wollen, hinterlegt ist, exakt dem Namen entspricht, den sie im Buchungssystem angegeben haben. Ist das nicht der Fall, können sie das entsprechende Zertifikat in der Corona-Warn-App nicht auswählen. Haben Sie ihrem Zertifikat beispielsweise ihren ersten und zweiten Vornamen hinterlegt, sollten sie beide Namen auch bei der Ticketbuchung angeben. Geben sie nur einen Namen an, kann die Corona-Warn-App nicht erkennen, dass es sich um ein und dieselbe Person handelt.",

```

The capitalization in this paragraph is wrong in many places.

```suggestion
                            "Sie sollten unbedingt darauf achten, dass der Name, der im Zertifikat, das Sie zur Überprüfung benutzen wollen, hinterlegt ist, exakt dem Namen entspricht, den Sie im Buchungssystem angegeben haben. Ist das nicht der Fall, können Sie das entsprechende Zertifikat in der Corona-Warn-App nicht auswählen. Haben Sie Ihrem Zertifikat beispielsweise Ihren ersten und zweiten Vornamen hinterlegt, sollten Sie beide Namen auch bei der Ticketbuchung angeben. Geben Sie nur einen Namen an, kann die Corona-Warn-App nicht erkennen, dass es sich um ein und dieselbe Person handelt.",
```